modules/ztp-creating-the-policygentemplate-cr.adoc

```diff
@@ -26,6 +26,23 @@ Depending on the specific requirements of your clusters, you might need more tha
 ====
 
 . Define all the policy namespaces in a YAML file similar to the example `out/argocd/example/policygentemplates/ns.yaml` file.
++
+[IMPORTANT]
+====
+Ensure that the namespaces meets the following requirements:
+
+. Namespaces must be prefixed with `ztp`. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ztp-sno-1-site
```

Comment:
We point the user to the examples in ns.yaml. Maybe pull one of those for the example here, eg
```
apiVersion: v1
kind: Namespace
metadata:
  name: ztp-common
```